src/CommunityToolkit.Maui/Views/Popup/Popup.shared.cs

```diff
@@ -30,17 +30,18 @@ public partial class Popup : ContentView
 	/// <summary>
 	///  Backing BindableProperty for the <see cref="CanBeDismissedByTappingOutsideOfPopup"/> property.
 	/// </summary>
-	public static readonly BindableProperty CanBeDismissedByTappingOutsideOfPopupProperty = BindableProperty.Create(nameof(CanBeDismissedByTappingOutsideOfPopup), typeof(bool), typeof(Popup), PopupDefaults.CanBeDismissedByTappingOutsideOfPopup);
+	public static readonly BindableProperty CanBeDismissedByTappingOutsideOfPopupProperty = BindableProperty.Create(nameof(CanBeDismissedByTappingOutsideOfPopup), typeof(bool), typeof(Popup), Options.DefaultPopupSettings.CanBeDismissedByTappingOutsideOfPopup);
 
 	/// <summary>
 	/// Initializes Popup
 	/// </summary>
 	public Popup()
 	{
-		Margin = PopupDefaults.Margin;
-		Padding = PopupDefaults.Padding;
-		HorizontalOptions = PopupDefaults.HorizontalOptions;
-		VerticalOptions = PopupDefaults.VerticalOptions;
+		Margin = Options.DefaultPopupSettings.Margin;
```

Comment:
I have dragged my heels reviewing this to see if I could find a clean way of avoiding setting these values in the constructor. I had hoped to override more of the `BindableProperty` definitions above in order to control the default values. Sadly I could only do it with providing more of a definition of the `MarginProperty` etc. which introduces more risk of things breaking in the future so I think we should just go ahead with this and see how it plays with styling in the next PR.